static/styles/lib/base.scss

```diff
@@ -4,7 +4,7 @@
 @import "./tether/tether.min.css";
 @import "./bootstrap/scss/bootstrap-flex";
 $fa-font-path: "/vendor-optimized/font-awesome/fonts";
-@import "font-awesome/scss/font-awesome.scss";
+@import "../../../node_modules/font-awesome/scss/font-awesome.scss";
```

Comment:
../../ is not a proper way to get to node_modules
why changing this?

Reply:
try
@import "~font-awesome/scss/font-awesome";
(i have no time sry)

Reply:
it should if webpack would be used for that, tried some variants but they didn't work.

Reply:
You are right!
https://stackoverflow.com/questions/41647984/importing-sass-partials-from-node-modules
with importPaths it should work

@adrianjost maybe you can doublecheck